connection.go

```diff
@@ -19,6 +19,7 @@ import (
 )
 
 const requestsMap = 128
+const defaultStreamId = 0
```

Comment:
It will be better something like that:
```suggestion
const ignoreStreamId = 0
```

https://www.tarantool.io/en/doc/latest/dev_guide/internals/box_protocol/#box-protocol-streams

If IPROTO_STREAM_ID equals zero the server instance will ignore it.